page/ajax/working-with-jsonp.md

```diff
@@ -5,30 +5,30 @@
 	"attribution": [ "jQuery Fundamentals" ]
 }</script>
 
-The advent of JSONP — essentially a consensual cross-site scripting hack — has opened the door to powerful mashups of content. Many prominent sites provide JSONP services, allowing you access to their content via a predefined API. A particularly great source of JSONP-formatted data is the [Yahoo! Query Language](http://developer.yahoo.com/yql/console/), which we'll use in the following example to fetch news about cats.
+The advent of JSONP — essentially a consensual cross-site scripting hack — has opened the door to powerful mashups of content. Many prominent sites provide JSONP services, allowing you access to their content via a predefined API. A particularly great source of JSONP-formatted data is the [Wikipedia API](https://en.wikipedia.org/w/api.php?action=help&modules=query), which we'll use in the following example to fetch a list of articles about [JavaScript libraries](https://en.wikipedia.org/wiki/Category:Free_software_programmed_in_JavaScript).
 
 ```
-// Using YQL and JSONP
 $.ajax({
-	url: "http://query.yahooapis.com/v1/public/yql",
+	// Tell MediaWiki what we want and that we want JSON
+	url: "https://en.wikipedia.org/w/api.php",
+	data: {
+		format: "json",
+		action: "query",
+		list: "categorymembers",
+		cmtitle: "Category:Free_software_programmed_in_JavaScript",
+		cmprop: "title",
+		cmlimit: 500
+	},
 
-	// The name of the callback parameter, as specified by the YQL service
+	// Name of the query parameter that jQuery will add, to set the callback function
 	jsonp: "callback",
 
 	// Tell jQuery we're expecting JSONP
 	dataType: "jsonp",
 
-	// Tell YQL what we want and that we want JSON
-	data: {
-		q: "select title,abstract,url from search.news where query=\"cat\"",
-		format: "json"
-	},
-
-	// Work with the response
-	success: function( response ) {
-		console.log( response ); // server response
-	}
+}).then(function( response ) {
```

Comment:
Can we also update the whole example to use the jQuery code style? Right now it uses it only partially (too few spaces).